docs/docs/40-contributor-guide/30-code-of-conduct.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Code of conduct
-description: Code of conduct
+sidebar_label: Code of conduct
+description: Contributor Code of Conduct for the open source Kargo project
 ---
 
-# Contributor Covenant Code of Conduct
+# Kargo Contributor Code of Conduct
```

Comment:
I am hesitant about changing the title here. [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/) is a widely recognized code of conduct used by many OSS projects. I believe that by keeping the original title, the title alone conveys the exact set of rules we that contributors are agreeing to play by.